FP_layout/app/src/main/java/com/example/dhyatmika/fp_layout/AppConfig.java

```diff
@@ -21,4 +26,26 @@ public static String getRegister() {
     public static String getLogout(String token) {
         return domain + logout + "?token=" + token;
     }
+
+    public static String getValidate(String token) {
+        return domain + validate + "?token=" + token;
+    }
+
+    public static String getStations() {
+        return domain + stations;
+    }
+
+    public static String getNearby(Double lat, Double lon) {
+        String s1 = Double.toString(lat);
```

Comment:
use descriptive name instead of `s1`, `s2, for example `latString`, `lotString` would read better